src/parser/box.py

```diff
@@ -342,6 +348,52 @@ def set_box_files(self, element):
         content += "</ul>"
         self.content = content
 
+    def set_box_snippets(self, element):
+        """set the attributes of a snippets box"""
+
+        snippets = element.getElementsByTagName("snippetListList")[0].getElementsByTagName("snippetList")
+
+        for snippet in snippets:
+            title = Utils.get_tag_attribute(snippet, "title", "jahia:value")
+            subtitle = Utils.get_tag_attribute(snippet, "subtitle", "jahia:value")
+            description = Utils.get_tag_attribute(snippet, "description", "jahia:value")
+            image = Utils.get_tag_attribute(snippet, "image", "jahia:value")
+            big_image = Utils.get_tag_attribute(snippet, "bigImage", "jahia:value")
+            enable_zoom = Utils.get_tag_attribute(snippet, "enableImageZoom", "jahia:value")
+
+            # fix the path for image
+            if "/files" in image:
+                image = image[image.rfind("/files"):]
```

Comment:
Peut-être que ce n'est pas une bonne idée de faire ce remplacement au vu du code se trouvant à la ligne 279 de `wp_exporter.py`, à savoir :
> if "/files" not in file.path:
return

Du coup, le fait de faire le "nettoyage" de `/files` dans le parsing de la Box fera que le lien ne sera jamais reconstruit lors de l'import

Reply:
"/files" est toujours présent, j'enlève juste ce qui vient avant:

            print("before:" + image)

            # fix the path for image
            if "/files" in image:
                image = image[image.rfind("/files"):]

            print("after: " + image)

Donne:

before:/content/sites/developpementdurable/files/60_etudes-recherche/vignette_chaires.jpg
after: /files/60_etudes-recherche/vignette_chaires.jpg

Reply:
Ups, mal compris ton code alors, désolé.